tests/integration/commands/test_get.py

```diff
@@ -1,11 +1,55 @@
+import random
+from string import ascii_letters
+
+import confluent_kafka
 import pytest
 from click.testing import CliRunner
 
-from esque.cli.commands import get_topics
+from esque.cli.commands import get_topics, get_consumergroups, get_brokers
+from esque.controller.topic_controller import TopicController
+from esque.resources.topic import Topic
+
+
+@pytest.mark.integration
+def test_smoke_test_get_topics(non_interactive_cli_runner: CliRunner):
+    result = non_interactive_cli_runner.invoke(get_topics, ["--verbose"])
+    assert "No config provided in" not in result.output
```

Comment:
checking for `result.exit_code == 0` should be sufficient. If anything, you should make sure that no exception whatsoever was thrown by `assert result.exception is None`.